core/node/crypto/blockchain.go

```diff
@@ -48,6 +50,7 @@ type Blockchain struct {
 	Config          *config.ChainConfig
 	InitialBlockNum BlockNumber
 	ChainMonitor    ChainMonitor
+	Metrics         infra.MetricsFactory
```

Comment:
Serge, did you ever set this value on Blockchain initialization?

Reply:
yes, in line 122 below